scripts/image_build.sh

```diff
@@ -115,3 +115,10 @@ curl https://raw.githubusercontent.com/sql-machine-learning/sqlflow/develop/exam
 
 # 9. install xgboost-launcher
 pip install xgboost-launcher==0.0.3
+
+# 10. install hadoop
```

Comment:
install Hadoop to use as the client when writing CSV to hive tables.

Reply:
done.